docs/c_cxx/Doxyfile

```diff
@@ -843,7 +843,7 @@ WARN_NO_PARAMDOC       = YES
 # Possible values are: NO, YES and FAIL_ON_WARNINGS.
 # The default value is: NO.
 
-WARN_AS_ERROR          = YES
+WARN_AS_ERROR          = NO
```

Comment:
Were there a bunch of warnings that we can't easily fix?

Reply:
We've fixed these now